recipes/gtest/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cmake $SRC_DIR/CMakeLists.txt
+make
+cp libgtest.a $PREFIX/lib/
+cp libgtest_main.a $PREFIX/lib/
```

Comment:
There is a way to get it to, and I really think that this should.  Please see the other PR.